keep/providers/slack_provider/slack_provider.py

```diff
@@ -103,14 +103,17 @@ def oauth2_logic(**payload) -> dict:
 
         return new_provider_info
 
-    def _notify(self, message="", blocks=[], channel="", **kwargs: dict):
+    def _notify(
+        self, message="", blocks=[], channel="", slack_timestamp="", **kwargs: dict
+    ):
         """
         Notify alert message to Slack using the Slack Incoming Webhook API
         https://api.slack.com/messaging/webhooks
 
         Args:
             kwargs (dict): The providers with context
         """
+        notify_data = None
         self.logger.info(
             f"Notifying message to Slack using {'webhook' if self.authentication_config.webhook_url else 'access token'}",
```

Comment:
Is it not possible to edit a message using webhooks? Can't you send ts in here as well?

Reply:
@talboren posting to the webhook only returns the 'ok' as the response. Timestamp is necessary to edit a message.

Reply:
There's no direct way but we can try

![Screenshot_2024-08-18-14-53-43-35_e4424258c8b8649f6e67d283a50a2cbc.jpg](https://github.com/user-attachments/assets/9f39559f-bd17-4054-ad34-d2dea86934e4)